index.html

```diff
@@ -5,44 +5,223 @@
+                        Stories</h1>
+                    <ul><span>CANOPY</span>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="about-project">A daily selection of privately
+                        personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="buttons">
+                        <ul>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                        </ul>
+                    </div>
+                    <button class="see-project"> See Project</button>
+                </div>
+            </div>
+        </section>
 
+        <section class="project-3 project">
+            <div class="container">
+                <img src="images/Snapshoot Portfolio (2).png">
+                <div class="client">
+                    <h1 class="client-name">Tonic</h1>
+                    <ul><span>CANOPY</span>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="about-project">A daily selection of privately
+                        personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="buttons">
+                        <ul>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                        </ul>
+                    </div>
+                    <button class="see-project"> See Project</button>
+                </div>
+            </div>
+        </section>
+
+        <section class="project-4 project">
+            <div class="container">
+                <img src="images/Snapshoot Portfolio (3).png">
+                <div class="client">
+                    <h1 class="client-name">Multi-Post
+                        Stories</h1>
+                    <ul><span>CANOPY</span>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="about-project">A daily selection of privately
+                        personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="buttons">
+                        <ul>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                        </ul>
+                    </div>
+                    <button class="see-project"> See Project</button>
+                </div>
+            </div>
+        </section>
+    </section>
+    <section class="about-myself second-part">
+        <section class="headline">
+            <h1 class="greeting">About Myself
+            </h1>
```

Comment:
- [ ] The whole `about` section as stated in the requirements had to be laid out using `flexbox`, kindly implement that.